packages/tools/client-debugger/client-debugger-view/src/ContainerHistoryLog.tsx

```diff
@@ -114,8 +114,15 @@ export function ContainerHistoryLog(props: ContainerHistoryLogProps): React.Reac
 							}}
 						>
 							<TableCell>
-								{getStateIcon(item.newState)}
-								{item.newState}
+								<span
```

Comment:
I take it Stack (horizontal) and StackItem didn't do the trick here? (Feel free to close with no action).

Reply:
Resolved by creating a separate `StackItem` and applying stylings to the items.